circuit_knitting/cutting/cutting_decomposition.py

```diff
@@ -294,9 +294,6 @@ def find_cuts(
             - seed: A seed for the pseudorandom number generator used by the optimizer
         constraints: Options for specifying the constraints for circuit cutting
 
-            - qubits_per_QPU: The maximum number of qubits each subcircuit can contain
```

Comment:
WHy did we delete this again?

Reply:
Oops, that was a mistake on my part. Will fix this.